src/clients/RTorrent.ts

```diff
@@ -451,6 +461,60 @@ export default class RTorrent implements TorrentClient {
 		await this.methodCallP<void>("d.check_hash", [infoHash]);
 	}
 
+	async resumeInjection(
+		infoHash: string,
+		decision: DecisionAnyMatch,
+		options: { checkOnce: boolean },
+	): Promise<void> {
+		let sleepTime = resumeSleepTime;
+		const maxRemainingBytes = getMaxRemainingBytes(decision);
+		const stopTime = getResumeStopTime();
+		let stop = false;
+		while (Date.now() < stopTime) {
+			if (options.checkOnce) {
+				if (stop) return;
+				stop = true;
+			}
+			await wait(sleepTime);
```

Comment:
these waits are going to be a bit wonky on run-once entrypoints like `cross-seed search` because they will create "holds" on the event loop which will make the program look like it's hanging. We could get around that by adding `.unref()` to the setTimeout call, but it would mean either the last 0-~1 torrents in a `cross-seed search` run don't get resumed, or we would turn it off entirely for `cross-seed search`. 

Having thought about this as I was writing the above, that stuff doesn't seem like a huge deal. But we also have less-than-ideal behavior on restart because we lose some of the program's state. Together these concerns kind of make me think—what would it take to run this as a job?

Reply:
I think it's unnecessary to run as a job since it's already covered by the inject job. For `cross-seed search` and `cross-seed rss`, it will seem like it hangs but it most cases it should recheck within a couple minutes. Unless there is tons of partial matches being rechecked.

For the inject job though, there is a log stating it's waiting for tasks to finish.